woocommerce/woocommerce
# Add Facebook to Recommended Step in OBW

### Changes proposed in this Pull Request:

Add the Facebook for WooCommerce recommendation in Recommended step in the onboarding wizard.

Closes #22198

### How to test the changes in this Pull Request:

Go to to recommended step in the onboarding wizard: `wp-admin/index.php?page=wc-setup&step=recommended`

You will see something like this:

<img width="770" alt="facebook-in-recommneded-step" src="https://user-images.githubusercontent.com/11487924/50246894-7ee57e80-03a4-11e9-858f-c505df138c4e.png">

### To do

- [x] verify we have the correct plugin slug
- [x] verify end-date

### Changelog entry

> Enter a short summary of all changes on this Pull Request. This will appear in the changelog if accepted.

Recommend Facebook for WooCommerce in setup wizard.
